src/sqlfluff/core/default_config.cfg

```diff
@@ -43,6 +43,7 @@ comma_style = trailing
 allow_scalar = True
 single_table_references = consistent
 unquoted_identifiers_policy = all
+group_by_and_order_by_style = consistent
```

Comment:
Should this be a rule-specific config? Can't see it being needed by other rules.

Reply:
moved to the rule specific section